src/main/kotlin/frc/robot/lib/MarkerCommands.kt

```diff
@@ -0,0 +1,18 @@
+package frc.robot.lib
+
+import edu.wpi.first.wpilibj.Alert
+import edu.wpi.first.wpilibj.DataLogManager
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+
+const val ABNORMAL_EVENT_NAME = "PROBLEM!"
+val alert = Alert(ABNORMAL_EVENT_NAME, Alert.AlertType.kWarning)
+
+fun markEvent(eventName: String): Command = Commands.runOnce({
+    DataLogManager.log(eventName)
+    alert.set(true)
+}).withTimeout(5.0).andThen(Commands.runOnce({
+    alert.set(false)
```

Comment:
Is this just for logging purposes? If so, will an alert be shown to the drivers?

Reply:
This is for logging purposes. The driver calls this command when there's issues and it'll save the timestamp.

Reply:
I wonder if it might distract the driver. Because we do want to use the alerts minimally so the drivers will actually look at them when critical errors happen.

Reply:
If we open the timestamp in advantagescope we can see when it exists and when it doesn't

Reply:
Okay. Still I'm not sure why you need this as an alert? Why is using `log()` not sufficient?
If the problem is that multiple messages will overlap, you can just add the timestamp or an index number to the message.

Reply:
When we do it as alert we can make it show up in advantagescope. If we log it we'll have to search through the logs for when it happend.

Reply:
1. You still log it
2. The messages log is used by team code only, it shouldn't have other types of messages.
3. Errors and warnings highlighted by AdvantageScope.
3. You can use another message entry for the driver markers.

I'm aware of the alert feature 🙂 but this is not the use case here, they should only be used to notify the drivers.